src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-sonarjs/use-primitive-type.ts

```diff
@@ -0,0 +1,32 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertUsePrimitiveType: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "no-new-wrappers",
+            },
+            {
+                ruleArguments: [
+                    {
+                        types: {
+                            String: {
+                                message: "Use string instead",
+                                fixWith: "string",
+                            },
+                            Boolean: {
+                                message: "Use boolean instead",
+                                fixWith: "boolean",
+                            },
+                            Number: {
+                                message: "Use number instead",
+                                fixWith: "number",
+                            },
+                        },
+                    },
+                ],
+                ruleName: "@typescript-eslint/ban-types",
```

Comment:
> Also in the new converter `convertUsePrimitiveType` I use the ESLint rule `@typescript-eslint/ban-types` with the types `String`, `Boolean`, and `Number`. Should I do this?

Indeed, this looks good. The [existing `ban-types` merger](https://github.com/typescript-eslint/tslint-to-eslint-config/blob/34484ada576cbc843ff5f4e4f3fd94b4239ff6c5/src/converters/lintConfigs/rules/ruleMergers/ban-types.ts) can handle opposing `ruleArguments.types` -- it merges them together.

It is missing logic for the `fixWith`s though: https://github.com/typescript-eslint/tslint-to-eslint-config/issues/1087